client_end_user/shared/src/commonMain/kotlin/di/NetworkModule.kt

```diff
@@ -25,10 +25,9 @@ val networkModule = module {
             }
 
             defaultRequest {
-
                 header("Content-Type", "application/json")
+                header("Application-Id", "1000")
```

Comment:
I guess we should remove this line from here. Develope now has this line in create use endpoint please pull it into ur branch and test again @ahmednasserzaza

Reply:
Done , and conflect with develop resolved 
thank you